packages/manager/modules/pci/src/projects/project/instances/instance/instance.component.js

```diff
@@ -6,38 +6,40 @@ export default {
   controller,
   bindings: {
     disablePrivateNetworks: '<',
-    projectId: '<',
-    instanceId: '<',
     instance: '<',
+    instanceId: '<',
     instancePrice: '<',
+    projectId: '<',
 
-    instanceLink: '<',
     consoleLink: '<',
     currentActiveLink: '<',
+    instanceLink: '<',
 
+    applicationAccess: '<',
+    attachVolume: '<',
+    createBackupInstance: '<',
+    deleteInstance: '<',
     editInstance: '<',
     enableMonthlyBillingInstance: '<',
-    createBackupInstance: '<',
-    startRescueInstance: '<',
     endRescueInstance: '<',
-    softRebootInstance: '<',
+    goToBlockStorages: '<',
     hardRebootInstance: '<',
-    startInstance: '<',
-    stopInstance: '<',
     reinstallInstance: '<',
     resumeInstance: '<',
-    deleteInstance: '<',
-    goToBlockStorages: '<',
-    applicationAccess: '<',
-    attachVolume: '<',
+    shelveInstance: '<',
+    softRebootInstance: '<',
+    startInstance: '<',
+    startRescueInstance: '<',
```

Comment:
Was better to group by feature : `startRescueInstance/endRescueInstance`...

Reply:
Updated.